frontend/pages/register.vue

```diff
@@ -0,0 +1,261 @@
+<script setup lang="ts">
+import { useField, useForm } from 'vee-validate'
+import { REGEXP_VALID_EMAIL } from '~/utils/regexp'
+import { Target } from '~/types/links'
+import { tOf } from '~/utils/translation'
+import { API_PATH } from '~/types/customFetch'
+
+const { t: $t } = useI18n()
+const { fetch } = useCustomFetch()
+const toast = useBcToast()
+
+useBcSeo('login_and_register.title_register')
+
+const { handleSubmit, errors } = useForm()
+const { value: email } = useField<string>('email', validateAddress)
+const { value: password1 } = useField<string>('password1', validatePassword1)
+const { value: password2 } = useField<string>('password2', validatePassword2)
```

Comment:
**Minor:** `password1`, `password2`, `validatePassword1`, `validatePassword2`, `password1Error` and `password2Error` are not descriptive. Consider using more meaningful names that clearly indicate the purpose.

Reply:
Good idea gain.

Fixed.